src/clr-angular/utils/focus/key-codes.enum.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum KeyCodes {
+  Home = 'Home',
+  End = 'End',
+  ArrowUp = 'ArrowUp',
+  ArrowDown = 'ArrowDown',
+  ArrowRight = 'ArrowRight',
+  ArrowLeft = 'ArrowLeft',
+  Enter = 'Enter',
+  Space = 'Space',
+  Tab = 'Tab',
+}
+
+export enum IEKeyCodes {
+  ArrowUp = 'Up',
```

Comment:
Unfortunately IE the key codes are different so I added this to make it more explicit.